scanapi/evaluators/string_evaluator.py

```diff
@@ -15,7 +15,7 @@ class StringEvaluator:
     )  # ${<variable>}
 
     @classmethod
-    def evaluate(cls, sequence, vars):
+    def evaluate(cls, sequence, vars, is_a_test_case=False):
```

Comment:
this `is_a_test_case` looks kind of scattered all over

Reply:
@winstonf88 Hmmm indeed... Any ideas of how to minimize it?

Reply:
I tried to isolate it a bit more in a separated `evaluate_assertion` method, but we can do it better in the future for sure.

Reply:
Really cool! it's much more isolated. Great job.

Reply:
Thanks, man 💚